storybook/src/elements/nuxeo-data-table/nuxeo-data-table-structure.js

```diff
@@ -2,8 +2,10 @@ import { html } from 'lit-html';
 import '@nuxeo/nuxeo-ui-elements/nuxeo-data-table/iron-data-table.js';
 import '@nuxeo/nuxeo-ui-elements/nuxeo-document-thumbnail/nuxeo-document-thumbnail.js';
 import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date.js';
-import '../../../../ui/widgets/nuxeo-date-picker';
-import '../../../.storybook/i18n';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-user-tag.js';
+import '../../../.storybook/routing.js';
+import '../../../.storybook/i18n.js';
```

Comment:
The import ordering is not correct.
Anyway, we should automate this, otherwise it is PITA

Reply:
Indeed.